app/services/candidate_interface/submit_application_choice.rb

```diff
@@ -23,13 +23,18 @@ def call
         ApplicationStateChange.new(application_choice).send_to_provider!
 
         SendNewApplicationEmailToProvider.new(application_choice:).call
-        CandidateMailer.application_choice_submitted(application_choice).deliver_later
+        CandidateMailer.application_choice_submitted(application_choice).deliver_later(wait: wait_time)
       end
     end
 
     def current_time
       Time.zone.now
     end
+
+    def wait_time
+      rand(2..15).minutes
```

Comment:
The PR description says 1..15 minutes. Not a blocker though

Reply:
Ugh, 2 is a typo. I'll fix that.